internal/server/server.go

```diff
@@ -30,6 +30,10 @@ import (
 	"google.golang.org/protobuf/encoding/protojson"
 )
 
+const (
+	GRPCMaxClientSendSize = 32 << 20
```

Comment:
should we make it configurable as well ? wdyt?

Reply:
Yes. I think we should be able to configure it also if needed.

Reply:
made both values to be configurable